### ISSUE TITLE
Ombi newsletter not showing recently movies, only TV shows

### ISSUE THREAD

**Issue by mxglory** [tidusjar/Ombi#3048]
**Describe the bug**
I set up the Ombi newsletter to be sent out every Monday at 8am. Today at 8am, I got it but it only showed recently added TV shows. The movies section was empty; didn't even have a title for it. Has anyone experienced this bug? I checked the logs and it says that there are zero movies that were sent out. All my movies are being added via Ombi requests and Radarr and are being marked available when it scans Plex for new additions, so not sure what's going on.

Even did a test of the newsletter and the test doesn't show any movies either; but it does show the title for it. Maybe a bug in this version (3.0.4256)?

**To Reproduce**
1. Setup Newsletter
2. Test

**Expected behavior**
During the Test or the scheduled time of delivering, recently added movies and TV shows should be sent out and included in the newsletter

**Screenshots**
Test newsletter:
![image](https://user-images.githubusercontent.com/28024839/60469111-7520bb80-9c10-11e9-9b5d-98ccbbe95c0f.png)

**Logs (Logs directory where Ombi is located)**
2019-07-01 08:05:57.322 -07:00 [Information] Plex Movies to send: 0
2019-07-01 08:05:57.364 -07:00 [Information] Emby Movies to send: 0
2019-07-01 08:05:57.365 -07:00 [Information] Albums to send: 0
2019-07-01 08:06:08.353 -07:00 [Information] Plex Episodes to send: 0
2019-07-01 08:06:08.353 -07:00 [Information] Emby Episodes to send: 0
2019-07-01 08:06:08.931 -07:00 [Error] StatusCode: NotFound, Reason: Not Found, RequestUri: http://api.themoviedb.org/3/tv/?api_key=b8eabaf5608b88d0298aa189dd90bf00

**Desktop (please complete the following information):**
 - unRAID

**Ombi Version (please complete the following information):**
 - Version 3.0.4256
- Media Server: binhex-plex Version 1.16.0.1226

**Comment by tidusjar** [comment]
That's very strange. I've not seen that before. Can you give the develop version a go? Lots of bugs have been fixed especially regarding Plex sync

**Comment by mxglory** [comment]
> That's very strange. I've not seen that before. Can you give the develop version a go? Lots of bugs have been fixed especially regarding Plex sync

I'll try that; though I am unfamiliar in installing the develop version on unRAID via the repositories. As of right now, I have 9 hours until my parity check is finished, then I need to do a reboot to install the latest version of unRAID (4.7.2). Maybe updating unRAID will fix the issue. I'll report back when the update is installed. 

FYI, I searched through the linuxserver.io Ombi post and someone is having the exact same issues as of Friday, June 28th at 6:33am. When I set up the Ombi newsletter last Sunday (6/23/19) and tested it, everything was working, including the scheduled one that was sent on Monday (6/24/19). So something happened between Monday afternoon and Friday morning I guess.

**Comment by aptalca** [comment]
Can confirm on the stable linuxserver image.
Ombi version 3.0.4256

I did a test of the newsletter for the first time, got a test email with just a tv show, no movies.

Then enabled the newsletter, set the permission for my user to receive newsletter and triggered (first time ever). Received an email with a ton of tv shows, no movies.

Updated the database beforehand.

**Comment by mxglory** [comment]
Just tried the develop version. Same issue. Only shows TV shows during the trigger and test; no movies.

**Comment by mxglory** [comment]
Update: I've tried the develop version; same issue. It looks like an update was pushed out last night, so Ombi is now running version 3.0.4654, but the issue is still there. Newsletter is only showing TV shows, no movies. Same as the screenshot in my OP. In addition, unRAID was updated to version 6.7.2.

**Comment by tidusjar** [comment]
Ok, interesting. So does it show the movies title like in the OP or just no movies what-so ever?

Can you go into the Plex settings and manually run the sync job? 

Are new movies you added to your media server appearing as available in the search?

**Comment by mxglory** [comment]
It shows the Movies title, just no actual movie posters.

I ran the manual sync in the Plex settings, issue still there.

Requested movies are being downloaded, moved, and marked as available. There were 3 movies that were added yesterday and are on my plex and marked available but they are not showing during the test.

Let me know if you'd like me to try anything else.

**Comment by tidusjar** [comment]
Are you using the test button or are you actually triggering the newsletter?

**Comment by mxglory** [comment]
> Are you using the test button or are you actually triggering the newsletter?

Both. However, the trigger button doesn't always work for some reason. Haven't found any reason why it wouldn't.

**Comment by tidusjar** [comment]
so the test button will just grab top 10 from your database, it does not reflect your recently added media,

The trigger will execute the newsletter like the schedule, so if there is new content it will send it out, if there is no new content since the last newsletter then nothing will be sent (we don't want to send duplicates)

**Comment by mxglory** [comment]
> so the test button will just grab top 10 from your database, it does not reflect your recently added media,
> 
> The trigger will execute the newsletter like the schedule, so if there is new content it will send it out, if there is no new content since the last newsletter then nothing will be sent (we don't want to send duplicates)

I think I've figured out why it won't show movies then. The trigger not working makes sense since the newsletter was went out as scheduled yesterday morning. But I have used the trigger button with the 3 movies I added last night and it wasn't sent out, so that's strange.

As for the test button, I was under the impression that it sent out a combination of tv shows and movies (total of 10). In my test however, it only sends out 10 episodes of one show (just counted them). It seems to only grab 10 episodes of one show that I added a few weeks ago, nothing recent.

If this is the expected behavior, then I think there is no actual issue. Unless what I mentioned is totally wrong. If you confirm, I'll close this post.

**Comment by tidusjar** [comment]
So that is expected, the test button is only for testing that Ombi is communicating successfully with your email provider and grabbing media.

So on the schedule everything should be working correctly

**Comment by mxglory** [comment]
> So that is expected, the test button is only for testing that Ombi is communicating successfully with your email provider and grabbing media.
> 
> So on the schedule everything should be working correctly

Got it. One final test I did was disabling TV shows in the newsletter and trying the test button again. The results are similar to my OP, except no TV shows title which is to be expected. I thought that by disabling TV shows, the test would grab 10 movies, but it didn't; shows the title but no movies. Is that normal behavior as well?

In addition, when the newsletter was sent out yesterday morning per the schedule I set, it only had TV shows, and everything that was added for the previous week which is normal (had 200+ episodes; multiple shows). But it didn't have any of the movies that I added during the previous week (there should have been 14 movies).

I will keep the newsletter enabled just for myself and run it parallel with Tautulli's newsletter and compare results after a couple weeks.

**Comment by tidusjar** [comment]
So movies should be picked up.

Can you please add a movie to plex, run the plex scan (make sure ombi has picked up that it's available by searching for it, the View on Plex button should be present). Then Trigger the newsletter manually and provide the log files?

**Comment by mxglory** [comment]
Something is going on with Ombi. I added a movie, ran the Plex scan multiple times and it never marked the movie as available. Manually marking it available didn't display the View on Plex button either when doing a search for it. Trying to access the logs freezes notepad. The log file keeps growing, currently at 195MB now and rising slowly every couple minutes. I rebooted Ombi, same issue. Will need to investigate. The trigger button also didn't send anything.

**Comment by mxglory** [comment]
> So movies should be picked up.
> 
> Can you please add a movie to plex, run the plex scan (make sure ombi has picked up that it's available by searching for it, the View on Plex button should be present). Then Trigger the newsletter manually and provide the log files?

Finally was able to finish testing and waiting for Ombi to pick up recently added content. Here is what I did. I disabled TV shows in the newsletter settings, clicked Trigger, and I then received a newsletter with the movies that were added last week that _should_ have been included in the newsletter that was sent out yesterday morning. However, the test movies I added today as a test did not appear on the Trigger newsletter despite it being marked available in Ombi and the View in Plex button being visible.

I then waited a few minutes, scanned my Plex libraries, made sure the metadata was obtained, did another newsletter Trigger and the new movies I added today were in the email.

Looks like everything is working now. Turns out there was just a bit of confusion during the testing and how the newsletters are generated, but I think I got it now.

Thanks tidusjar for the assistance. I'll close this post.
